posts/2022-03-11-dev-mode-with-eclipse.adoc

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "Rapid cloud-native development in the Eclipse IDE with Liberty dev mode"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/kathrynkodama
+author_github: https://github.com/kathrynkodama
+seo-title: Rapid cloud-native development in the Eclipse IDE with Liberty dev mode - OpenLiberty.io
+seo-description: Liberty dev mode enables developers to rapidly develop cloud-native Java applications using MicroProfile and Jakarta EE in the Eclipse IDE.
+blog_description: "Liberty dev mode enables developers to rapidly develop cloud-native Java applications using MicroProfile and Jakarta EE in the Eclipse IDE."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Rapid cloud-native development in the Eclipse IDE with Liberty dev mode
+Kathryn Kodama <https://github.com/kathrynkodama>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Liberty dev mode enables developers to quickly and iteratively develop cloud-native Java applications with the latest Jakarta EE and MicroProfile technologies. Dev mode will watch your project for file changes and provide hot reload and deployment, on demand testing, and debugger support. You can work with dev mode from anywhere, whether it's from a terminal with a text editor or your favorite IDE (integrated development environment). This blog post details the current best practice for using dev mode to develop your application within the Eclipse IDE.
+
+Check out a demo of working with Liberty dev mode in the Eclipse IDE:
+++++
+<iframe width="560" height="315" src="https://youtu.be/613VBYdk6f8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+== Setting up the project
+
+For Maven projects, specify the Liberty Maven Plugin in your project's `pom.xml` file:
+[source,xml]
+----
+    <plugin>
+        <groupId>io.openliberty.tools</groupId>
+        <artifactId>liberty-maven-plugin</artifactId>
+        <version>3.5.1</version>
+    </plugin>
+----
+
+For Gradle projects, specify the Liberty Gradle Plugin in your project's `build.gradle` file:
+[source,groovy]
+----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.3'
+    }
+}
+----
+
+== Starting dev mode
+
+Start dev mode through the Eclipse IDE terminal ("Window" -> "Show View" -> "Terminal") or through your operating system's terminal with the following commands:
```

Comment:
When I just do this "Show view" way to access the terminal I don't get a command prompt.    OTOH, if I go to Package Explorer, right-click, "Show in Local Terminal" and then make a selection like "Terminal", I get a prompt right at the corresponding directory...which is more useful.